src/beach.ts

```diff
@@ -0,0 +1,730 @@
+import { cliExecute, myId } from "kolmafia";
+import { get, set } from "libram";
+
+type BeachTile = { minute: number; row: number; column: number };
+
+const rareTiles: BeachTile[] = [
+  { minute: 41, row: 4, column: 6 },
```

Comment:
Would it be better to store this data in a kolmafia data file? Or is that more work? I figure that adding new data to this list will be more painful if the data is in a code file.

Reply:
That is probably a better idea. I should read up on how to do that.